modules/oxide/package.json

```diff
@@ -40,6 +40,7 @@
   "devDependencies": {
     "connect": "^3.7.0",
     "connect-livereload": "^0.6.1",
+    "grunt-string-replace": "^1.3.3",
     "stylelint-config-recommended-less": "^3.0.1"
   }
-}
+}
```

Comment:
Not quite sure what the change is here, probably no newline at the end of file? Let's rather keep it as it was before.